.gitignore

```
@@ -19,10 +19,10 @@ doc/framework
 dead_links.json
 
 # Cypress debug
-doc/6/getting-started/.react/cypress/screenshots
-doc/6/getting-started/.react/cypress/videos
-doc/6/getting-started/.vuejs/cypress/screenshots
-doc/6/getting-started/.vuejs/cypress/videos
+doc/7/getting-started/.react/cypress/screenshots
+doc/7/getting-started/.react/cypress/videos
+doc/7/getting-started/.vuejs/cypress/screenshots
+doc/7/getting-started/.vuejs/cypress/videos
```

So this .gitignore has to be updated every time a new version comes out?
```suggestion
doc/*/getting-started/.react/cypress/screenshots
doc/*/getting-started/.react/cypress/videos
doc/*/getting-started/.vuejs/cypress/screenshots
doc/*/getting-started/.vuejs/cypress/videos
```